client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -26,40 +36,66 @@ export class GoogleMyBusiness extends SharingService {
 	};
 
 	// override `createOrUpdateConnection` to ignore connection update, this is only useful for publicize services
-	createOrUpdateConnection = () => {};
+	createOrUpdateConnection = ( keyringConnectionId, externalUserId ) => {
+		this.props
+			.connectGoogleMyBusinessLocation( this.props.siteId, keyringConnectionId, externalUserId )
+			.catch( () => {
+				this.props.failCreateConnection( {
+					message: this.props.translate( 'Error while linking your site to %(service)s.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?